src/lithium/interestingness/ximport.py

```diff
@@ -13,31 +13,29 @@
 import os
 import sys
 
-log = logging.getLogger("lithium")  # pylint: disable=invalid-name
 
-
-def importRelativeOrAbsolute(f):  # pylint: disable=invalid-name
+def rel_or_abs_import(f):
```

Comment:
I switched everything to use absolute imports instead of relative imports (e.g. `from lithium.interestingness import ximport` instead of `from . import ximport`) and it seems to work:

```
$ python -m lithium range.py /dev/null
Intermediate files will be stored in tmp3/.
The original testcase has 0 lines.
Checking that the original testcase is 'interesting'...
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/__main__.py", line 13, in <module>
    main()
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1416, in main
    exit(Lithium().main())
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1187, in main
    return self.run()
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1203, in run
    result = self.strategy.main(self.testcase, self.interesting, self.testcaseTempFilename)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 328, in main
    if not interesting(testcase, writeIt=False):
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1368, in interesting
    inter = self.conditionScript.interesting(self.conditionArgs, tempPrefix)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/interestingness/range.py", line 47, in interesting
    (range_min, range_max, arguments) = parse_options(cli_args)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/interestingness/range.py", line 43, in parse_options
    return int(args[0]), int(args[1]), args[2:]  # args[0] is minLoopNum, args[1] maxLoopNum
ValueError: invalid literal for int() with base 10: '/dev/null'
$ python -m lithium range /dev/null
Intermediate files will be stored in tmp4/.
The original testcase has 0 lines.
Checking that the original testcase is 'interesting'...
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/__main__.py", line 13, in <module>
    main()
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1416, in main
    exit(Lithium().main())
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1187, in main
    return self.run()
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1203, in run
    result = self.strategy.main(self.testcase, self.interesting, self.testcaseTempFilename)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 328, in main
    if not interesting(testcase, writeIt=False):
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/reducer.py", line 1368, in interesting
    inter = self.conditionScript.interesting(self.conditionArgs, tempPrefix)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/interestingness/range.py", line 47, in interesting
    (range_min, range_max, arguments) = parse_options(cli_args)
  File "/home/gkwubu/.local/lib/python2.7/site-packages/lithium/interestingness/range.py", line 43, in parse_options
    return int(args[0]), int(args[1]), args[2:]  # args[0] is minLoopNum, args[1] maxLoopNum
ValueError: invalid literal for int() with base 10: '/dev/null'
```
Note that there is no longer the `ValueError: Attempted relative import in non-package` error.

Would you like to retest to see if it is necessary to import your `ximport.py` changes?